src/pie_utils/processor/document/split_into_partition.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+import json
+import logging
+import re
+from typing import Any, Callable, Iterable, Iterator, Match
+
+from pytorch_ie.annotations import LabeledSpan, Span
```

Comment:
Is Span still used anywhere?

Reply:
Yeah this can be removed now, an extra variable from sanity check prevented it from being removed while optimizing imports.

Reply:
What does this mean?

Reply:
I mean while removing code related to sanity check in `split_into_partition.py `, I missed removing a variable of type `Span` that was being used in that code block. Its fixed in [ae085da](https://github.com/ArneBinder/pie-utils/pull/21/commits/ae085da985e50e43213cb7c97299bd4dc52e9c07).

Reply:
ah ok, thanks!